index.html

```diff
@@ -86,7 +93,23 @@ <h2> Hello I'm Yetzirah</h2>
             </div>
             <div class="section-content">
               <p>I am a Software developer with 2 years of experience in designing and developing web applications and also creating solutions using a popular ecommerce platform named Shopify.</p>
-              <div class="share-button">SHARE</div>
+              <div class="share-section">
+                <button onclick="showShareButtons()" class="share-options">SHARE</button>
+                <div class="share-button-container" style="display: none;">
+                  <a href="http://twitter.com/share?url=https://yetzirah.github.io/curriculum-vitae/&text=<TEXT>&via=<USERNAME>" target="_blank" class="share-button twitter">
```

Comment:
You should implement placeholderse `<TEXT> https://yetzirah.github.io/curriculum-vitae/ via @<USERNAME>`